### ISSUE TITLE
Using generated C# "struct" classes in a Xamarin Android app?

### ISSUE THREAD

**Issue by mwherman2000** [Microsoft/GraphEngine#339]
I'd like to be able to use the generated C# classes from a common set of TSL structs in both a Xamarin Android app and an ASP.NET/Blazor web app.

Has anyone tried this?  ....any suggestions?

**Comment by TaviTruman** [comment]
@mwherman2000 Hi did you ever get an answer to your question; I am looking into this myself, as I have a similar need.

**Comment by mwherman2000** [comment]
No, I ended up recoding the structures I needed in my Android client app.

**Comment by TaviTruman** [comment]
@mwherman2000 Yeah, I have done that myself; I've been experimenting with a few things to retain the generated code. I will add this capability to the Ideas section in the Discussions part of the repo. Thanks for the reply.

**Comment by mwherman2000** [comment]
@TaviTruman please do :-)

**Comment by mwherman2000** [comment]
Use Claude
